src/net.rs

```diff
@@ -143,29 +136,29 @@ impl NetworkStream + Send {
 }
 
 /// A `NetworkListener` for `HttpStream`s.
-pub enum HttpListener {
-    /// Http variant.
-    Http(TcpListener),
-    /// Https variant. The two paths point to the certificate and key PEM files, in that order.
-    Https(TcpListener, Arc<SslContext>)
-}
+pub struct HttpListener(TcpListener);
+
+/// Https variant. The two paths point to the certificate and key PEM files, in that order.
```

Comment:
Stray commented code related to this type. As far as I can see it can just be deleted.